src/crypto/CHIPCryptoPAL.h

```diff
@@ -621,27 +620,20 @@ class Symmetric128BitsKeyHandle
  */
 class Aes128KeyHandle final : public Symmetric128BitsKeyHandle
 {
-public:
-    Aes128KeyHandle() = default;
-
-    Aes128KeyHandle(const Aes128KeyHandle &) = delete;
-    Aes128KeyHandle(Aes128KeyHandle &&)      = delete;
-    void operator=(const Aes128KeyHandle &)  = delete;
-    void operator=(Aes128KeyHandle &&)       = delete;
 };
 
 /**
  * @brief Platform-specific HMAC key handle
  */
 class Hmac128KeyHandle final : public Symmetric128BitsKeyHandle
 {
-public:
-    Hmac128KeyHandle() = default;
+};
 
-    Hmac128KeyHandle(const Hmac128KeyHandle &) = delete;
-    Hmac128KeyHandle(Hmac128KeyHandle &&)      = delete;
-    void operator=(const Hmac128KeyHandle &)   = delete;
-    void operator=(Hmac128KeyHandle &&)        = delete;
+/**
+ * @brief Platform-specific HKDF key handle
```

Comment:
Why the specificity on 128 bits as the source of the key, with HKDF? Is it because a key in PSA must be marked "KDF-allowed" rather than just using HKDF against a standard 128 bit symmetric key handle (as opposed to an ECDH shared secret for CASE) ?

Reply:
When I introduced AES key handle I was asked to make a type for 128-bit key specifically, so this time I wanted to follow the pattern we already have.
I believe for raw-key implementations, having one type for all key lengths would mean that the key handle must be as big as the maximum key supported or it must store a pointer to a heap allocated key. This may be suboptimal solution for those implementations, but in PSA crypto API (which our platform is moving to shortly) all keys are represented by the same _key ID_ type, so I would definitely not object having fewer key handle types if everyone agrees on that :).
It's kinda true that HKDF is the first example in which the strategy we take makes a difference as we use 128-bit key for PASE and 256-bit key for CASE.